courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -305,36 +297,34 @@ Multiple Object Declarations
 Predefined Declarations
 -------------------------
 
-* Many items are already declared for users
+* **Implicit** declarations
+* Language standard
+* Annex A for *Core*
 
-   - Defined by language standard
-   - Defined by the implementation
+   - Package :code:`Standard`
+   - Standard types and operators
 
-* See Annex A for details of predefined declarations
+        + Numerical
+        + Characters
 
-   - Package named `Standard`
-   - Numeric types and operators
-   - Character types and string types
-   - Input/Output facilities
-   - String handling facilities
-   - Access to command line arguments
-   - Many, many others
+   - About **half the RM** in size
 
-* Half the RM describes the Standard Libraries
+* **Optional** in *Specialized Needs Annexes*
```

Comment:
Should follow same format as previous bullet ("Core in annex A" or "Specialized Needs Annexes for optional")